src/satosa/util.py

```diff
@@ -7,6 +7,10 @@
 
 logger = logging.getLogger(__name__)
 
+def get_dict_defaults(d, *keys):
+    for key in keys:
+       d = d.get(key, d.get("", d.get("default", {})))
+    return d
```

Comment:
No. I want to traverse the entire length of the "key chain" finding the most specific branch.